content/applications/hr/referrals.rst

```diff
@@ -471,7 +406,9 @@ spreadsheet from the drop-down menu, or select :guilabel:`New Spreadsheet`. Clic
 
 .. seealso::
    :doc:`referrals/share_jobs`
+   :doc:`referrals/rewards`
```

Comment:
Add bullets for multiple seealso items :)
```suggestion
   - :doc:`referrals/share_jobs`
   - :doc:`referrals/rewards`
```

Reply:
OMG I was used to removing the bullets when there was only one doc, I totally forgot!